src/Xamarin.Android.Build.Tasks/Microsoft.NET.Workload.Android/WorkloadManifest.in.json

```diff
@@ -0,0 +1,26 @@
+{
+  "version": 5,
+  "workloads": {
+    "microsoft-android-sdk-full": {
+      "description": "Android SDK",
+      "packs": [
+        "Microsoft.Android.Sdk",
+        "Microsoft.Android.Ref"
+      ]
+    }
+  },
+  "packs": {
+    "Microsoft.Android.Sdk": {
+      "kind": "sdk",
+      "version": "@SDK_PACK_VERSION@",
+      "alias-to": {
+        "osx-x64": "Microsoft.Android.Sdk.osx-x64",
+        "windows-x64": "Microsoft.Android.Sdk.windows-x64"
```

Comment:
We'll need to change this back to win-x86 once https://github.com/dotnet/sdk/pull/14393 lands.